### PR TITLE
feat(settings): implement view and unblock blocked peers

### PR DESCRIPTION
**Pull request by 24thsaint**
image

### CODE REVIEW

File: src/pages/Settings/ProfileSettings.tsx [hunk 1]
```diff
@@ -412,200 +395,223 @@ class GeneralProfile extends Component<ProfileSettings, GeneralProfileState> {
...
+                searchChange={this.searchCompetency}
+                searchResults={this.state.seachResultComp}
+                val={this.state.searhCompQuery}
+              />
+              <CompetencySelector
+                checker={this.toggleCompetency}
+                competencies={this.state.competencySelector.competencies}
+                showTest={this.showTest}
+              />
+            </>
```
Comment (jglosbanes) [inline src/pages/Settings/ProfileSettings.tsx]: Intentional ang extra set sang brackets?

File: src/components/Card/Settings/BlockedPeersCard.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+interface BlockedPeersCardProps {
+  settings: Settings
+}
+
+const BlockedPeersCard = (props: BlockedPeersCardProps) => {
+  const profiles: Profile[] = []
+  const [blockedPeersInformation, setBlockedPeersInformation] = useState(profiles)
+
+  useEffect(() => {
+    ;(async () => {
```
Comment (jglosbanes) [inline src/components/Card/Settings/BlockedPeersCard.tsx]: Intentional ang semicolon?

File: src/pages/Settings/ProfileSettings.tsx (hunk 1 above)
Comment (24thsaint) [reply src/pages/Settings/ProfileSettings.tsx]: Yes. :))

File: src/components/Card/Settings/BlockedPeersCard.tsx (hunk 2 above)
Comment (24thsaint) [reply src/components/Card/Settings/BlockedPeersCard.tsx]: Yes. Linter rules for IIFE.

**Pull request merged**